Library/Homebrew/tab.rb

```diff
@@ -250,6 +250,10 @@ def to_s
     when true  then s << "Poured from bottle"
     when false then s << "Built from source"
     end
+    unless time.nil?
+      s << "Installed" if s.empty?
+      s << Time.at(time).strftime("on %Y-%m-%d at %H:%M:%S")
```

Comment:
Default is something like `2016-05-05 11:57:27 -0700`, doesn't work well in a phrase or a sentence. As for localization, `Poured from bottle`/`Built from source` are not localized, so only localizing the datetime part would be pretty weird.